ctapipe/io/tests/test_zfits.py

```diff
@@ -0,0 +1,33 @@
+from pkg_resources import resource_filename
+import os
+
+example_file_path = resource_filename(
+    'protozfitsreader',
+    os.path.join(
+        'tests',
+        'resources',
+        'example_10evts.fits.fz'
+    )
+)
+
+FIRST_EVENT_NUMBER_IN_FILE = 97750287
+ADC_SAMPLES_SHAPE = (1296, 50)
+
+
```

Comment:
Can you add some tests to check `ZFitsEventSource` is correctly returned from `EventSourceFactory` for the relevant input file?

Reply:
Ah .. well sort of.  I do not think, this PR should test that EventSourceFactory works as expected. But you are right, `is_compatible` has not been tested.

Reply:
`is_compatible` was indeed broken and the test showed it nicely. Thank's for pointing that out

Reply:
Adding a test specifically for checking `ZFitsEventSource ` is returned from `EventSourceFactory` will help us to quickly identify problems such as where two different file formats are found to be compatible with the same `EventSource`, or a single file format is designated as compatible with two different `EventSource`s

Reply:
Ah .. I see ... I'll have to look at similar tests from other event sources to see how one can test for this.... I doubt it is possible.